Lesson9/Gokhan/index.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+  <html lang="en">
+<head>
+  <title>Responsive Images</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body class="text-content">
+  
+  <pre>
```

Comment:
I don't think using pre element make any sence here. https://www.w3schools.com/tags/tag_pre.asp
```suggestion
  <main class="text-content">
```